stl/inc/chrono

```diff
@@ -6291,8 +6293,7 @@ namespace chrono {
             }
         }
 
-        _NODISCARD static array<_CharT, 4> _Fmt_string(const _Chrono_spec<_CharT>& _Spec) {
-            array<_CharT, 4> _Fmt_str;
+        _NODISCARD static _CharT* _Fmt_string(const _Chrono_spec<_CharT>& _Spec, _CharT* _Fmt_str) {

```

Comment:
This is correct, but I believe that slight safety improvements are possible.

First, we're always writing into `_CharT _Fmt_str[4];`. If `_Fmt_string()` took `_CharT (&_Fmt_str)[4]`, then it would be impossible to accidentally pass a too-small buffer. (Since it's an internal helper, we don't need to generalize it further; if we needed to, `span<_CharT, 4>` would be possible.)

Second, this is passed to `put_time()` which just needs to read the format string, so we can return `const _CharT*` which makes the intended usage clearer (the function says "I need to be given an array to fill, and then I'll return a pointer for reading").

I'll validate and push changes.